src/file-system-cache-blob-storage.js

```diff
@@ -0,0 +1,93 @@
+var fs = require('fs-plus')
+var path = require('path')
+
+module.exports = (function () {
+  FileSystemCacheBlobStorage.load = function (directory) {
+    var instance = new FileSystemCacheBlobStorage(directory)
+    instance.load()
+    return instance
+  }
+
+  function FileSystemCacheBlobStorage (directory) {
+    this.inMemoryCache = new Map()
+    this.cacheBlobFilename = path.join(directory, 'v8-compile-cache.blob')
```

Comment:
Based on the above comment, I think these filenames need to be changed as well. I cannot think of anything more generic than: `BLOB` and `MAP`. Ideas?